Entity-ngsiv2-schema.json

```diff
@@ -0,0 +1,185 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/Entity-ngsiv2-schema.json",
+  "title": "NGSIv2 Entity",
+  "description": "NGSIv2 Entity",
+  "definitions": {
+    "EntityIdentifier": {
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 256,
+      "pattern": "^[\\w\\-\\.\\{\\}\\$\\+\\*\\[\\]`|~^@!,:\\\\]+$"
```

Comment:
Could you describe this pattern in "natural lenguage", pls? :)

Reply:
it was developed time ago but it matches the NGSIv2 specification

Reply:
From NGSIv2 spec:

> Allowed characters are the ones in the plain ASCII set, except the following ones: control characters, whitespace, `&`, `?`, `/` and `#`.

We should ensure that pattern matches exactly this. I'm not sure... it is a bit cryptic.

Maybe a ASCII range based (i.e. based in `\xNN` sintax) would be clearer?

Reply:
I think it is perfectly clear, RegExp are cryptic by themselves and you have to always check documentation

Reply:
NTC

Reply:
Something like `[\x20-xFF]` (just an example) is probably much less cryptic than the current one in the PR.

I'd suggest to do the following: let's see how is the equivalent regex in "the `\xNN` way" looks like, compare with the existing one, and choose. Makes sense?

Reply:
I'm not gonna change this for the sake of changing, it works, it has been there for a long time, no need to change. 

NTC